optuna/integration/botorch.py

```diff
@@ -542,9 +573,14 @@ def sample_relative(
         if len(search_space) == 0:
             return {}
 
-        trials = study.get_trials(deepcopy=False, states=(TrialState.COMPLETE,))
+        completed_trials = study.get_trials(deepcopy=False, states=(TrialState.COMPLETE,))
+        running_trials = [
+            t for t in study.get_trials(deepcopy=False, states=(TrialState.RUNNING,)) if t != trial
+        ]
```

Comment:
Just for your information, I was missing a detail here in the previous commit; the `running_trials` include the `trial` itself so I need to filter it out.